src/app/src/pages/eval/components/ColumnSelector.tsx

```diff
@@ -71,7 +71,7 @@ export const ColumnSelector: React.FC<ColumnSelectorProps> = ({
   // Group columns by their group property
   const groupedColumns = columnData.reduce(
     (acc, column) => {
-      const group = column.group || 'Other';
+      const group = column?.group || 'Other';
```

Comment:
Column being undefined caused the react app to crash on reload a few times. I am sneaking it in in this PR.